### ISSUE TITLE
ui5lint complains about the property `datetime` in NotificationListItem

### ISSUE THREAD

**Issue by mofterdinger** [SAP/ui5-linter#984]
Hi,

I'm currently getting this warning from the `ui5lint` tool:

```
  19:15 error Use of deprecated property 'datetime' of class 'NotificationListItem'. Details: As of version 1.123. this property is available directly on sap.m.NotificationListItem (https://ui5.sap.com/1.136/#/api/sap.m.NotificationListItem).  no-deprecated-api
```

## Expected Behavior

No warnings from the `ui5lint` about the property `datetime` in NotificationListItem. Regarding the current documentation this property is not deprecated: https://ui5.sap.com/1.143.0/#/api/sap.m.NotificationListItem%23controlProperties

## Current Behavior

```
UI5 linter report:

/Users/D032842/SAPDevelop/git/regatta-infopoint/static/webapp/view/Launchpad.view.xml
  19:15 error Use of deprecated property 'datetime' of class 'NotificationListItem'. Details: As of version 1.123. this property is available directly on sap.m.NotificationListItem (https://ui5.sap.com/1.136/#/api/sap.m.NotificationListItem).  no-deprecated-api

1 problem (1 error, 0 warnings)
   Run "ui5lint --fix" to resolve all auto-fixable problems
```

## Steps to Reproduce the Issue

1. git clone repo: https://github.com/Heidelberger-Regattaverband/regatta-infopoint
2. cd into folder `static`
3. perform command: `ui5lint`

## Context

- UI5 linter version: **`1.20.8`**
- Node.js version: **`v22.22.0`**
- npm version: `10.9.4`
- OS/Platform: `MacOS 26.2`
- Browser *(if relevant)*: `unknown`
- Other information regarding your environment *(optional)*: `nothing`

## Log Output / Stack Trace

```
{...}
```
